packages/datadog-plugin-jest/src/index.js

```diff
@@ -139,6 +142,28 @@ function createHandleTestEvent (tracer, testEnvironmentMetadata, instrumenter) {
       testSpan.finish()
       return
     }
+    if (event.name === 'hook_failure') {
+      const testSpan = tracer.startSpan(
```

Comment:
Anyway way to wrap the hook to also get the timing information? What if the hook timed out or took 10 minutes before failing for example? The also applies to the other test frameworks.

Reply:
yeah that's a good point. Would it be OK to do it in a follow up PR? This is getting quite big already and it already solves the problem of identifying errors in the hooks.

Reply:
there isn't one. For CI App, the trace will soon no longer be the entry point to a test, as we're seeing that most of the times (unit tests) we can only show a single span. Also, this trace will only have this span, so there'll be no problem with widths being too small in the UI.

In any case, I agree this needs to be solved. I just wonder if it'd be better to do it in a separate PR, so I don't have to keep rebasing /merge master on this PR.